utils/Main.py

```diff
@@ -0,0 +1,97 @@
+#!/bin/env python
+import http.client, pdb, socket, ssl, threading, select
+from time import (
+    sleep,
+)  # Needed to prevent busy-waiting for the browser to complete the login process!
+from flexclient.SmartLink import SmartLink
+from flexclient.Radio import Radio
+import flexclient.DataHandler
+import numpy, time
+from sys import exit
+import os
+import matplotlib.pyplot as plt
+from matplotlib.animation import FuncAnimation
+
+SERIAL = os.getenv("FLEX_SERIAL_NUMBER") or None

```

Comment:
os.getenv() does the "or None" defaulting by itself. If you want to be explicit about it, you can say os.getenv("FLEX_SERIAL_NUMBER", default=None)

Reply:
fixed